### ISSUE TITLE
Not able to delete a row from a cell component

### ISSUE THREAD

**Issue by GCorbel** [quantosobra/ember-data-table-light#3]
I have this simple component : 

```
import Ember from 'ember';

export default Ember.Component.extend({
  model: Ember.computed.alias('row.content'),

  actions: {
    delete(model) {
      this.get('tableActions').deleteUser(model);
    }
  }
})
```

I think to pass the table in component can be enough. We can also pass the datable in arguments of the data-table component.

I want to delete a row when I delete a record but I think I have no simple way to do it. I don't know if it's better to reload data or simple remove the row. What you think?

**Comment by eduardoweiland** [comment]
I think this is related to https://github.com/offirgolan/ember-light-table/issues/81

Maybe using the `row` property in your cell component, you can pass it to some action in `tableActions` and then use the [removeRow method](http://offirgolan.github.io/ember-light-table/docs/classes/Table.html#method_removeRow) in `Table`, but I haven't tested this.

**Comment by GCorbel** [comment]
I saw this solution but I don't have access to `Table`. There is no information about the table on the row or on the column.

**Comment by eduardoweiland** [comment]
Fixed in #4
